### PR TITLE
Update fr-FR.json

### PR DESCRIPTION
**Pull request by axgd-code**
Translation of new strings into French and corrections on existing strings.

### CODE REVIEW

File: mobile/assets/i18n/fr-FR.json [hunk 1]
```diff
@@ -268,8 +268,8 @@
   "map_zoom_to_see_photos": "Dézoomer pour voir les photos",
   "monthly_title_text_date_format": "MMMM y",
   "motion_photos_page_title": "Photos avec mouvement",
-  "multiselect_grid_edit_date_time_err_read_only": "Cannot edit date of read only asset(s), skipping",
-  "multiselect_grid_edit_gps_err_read_only": "Cannot edit location of read only asset(s), skipping",
+  "multiselect_grid_edit_date_time_err_read_only": "Impossible de modifier l'emplacement du ou des actifs en lecture seule, la modification est donc ignorée.",
+  "multiselect_grid_edit_gps_err_read_only": "Impossible de modifier l'emplacement du ou des actifs en lecture seule, la modification est donc ignorée.",
```
Comment (martabal) [inline mobile/assets/i18n/fr-FR.json:272]:
```suggestion
  "multiselect_grid_edit_date_time_err_read_only": "Impossible de modifier l'emplacement de médias en lecture seule, la modification est donc ignorée.",
  "multiselect_grid_edit_gps_err_read_only": "Impossible de modifier l'emplacement de médias en lecture seule, la modification est donc ignorée.",
```

File: mobile/assets/i18n/fr-FR.json [hunk 2]
```diff
@@ -296,32 +296,32 @@
...
   "profile_drawer_sign_out": "Se déconnecter",
   "profile_drawer_trash": "Corbeille",
   "recently_added_page_title": "Récemment ajouté",
-  "scaffold_body_error_occurred": "Error occurred",
+  "scaffold_body_error_occurred": "Une erreur s'est produite",
   "search_bar_hint": "Rechercher vos photos",
   "search_page_categories": "Catégories",
   "search_page_favorites": "Favoris",
-  "search_page_motion_photos": "Photos avec mouvement",
+  "search_page_motion_photos": "Photos en mouvement",
```
Comment (martabal) [inline mobile/assets/i18n/fr-FR.json:314]: `moving photo` != `motion photo`
Probably `photos animées` is better ?

File: mobile/assets/i18n/fr-FR.json [hunk 3]
```diff
@@ -296,32 +296,32 @@
...
-  "search_page_person_add_name_subtitle": "Find them fast by name with search",
-  "search_page_person_add_name_title": "Add a name",
-  "search_page_person_edit_name": "Edit name",
+  "search_page_person_add_name_dialog_cancel": "Annuler",
+  "search_page_person_add_name_dialog_hint": "Nom",
+  "search_page_person_add_name_dialog_save": "Enregistrer",
+  "search_page_person_add_name_dialog_title": "Ajouter un nom",
+  "search_page_person_add_name_subtitle": "Retrouver les rapidement via la recherche par nom",
+  "search_page_person_add_name_title": "Ajouter un nom",
+  "search_page_person_edit_name": "Modifier nom",
```
Comment (martabal) [inline mobile/assets/i18n/fr-FR.json:324]:
```suggestion
  "search_page_person_edit_name": "Modifier le nom",
```

File: mobile/assets/i18n/fr-FR.json [hunk 4]
```diff
@@ -142,21 +142,21 @@
   "control_bottom_app_bar_archive": "Archive",
   "control_bottom_app_bar_create_new_album": "Créer un nouvel album",
   "control_bottom_app_bar_delete": "Supprimer",
-  "control_bottom_app_bar_edit_location": "Edit Location",
-  "control_bottom_app_bar_edit_time": "Edit Date & Time",
+  "control_bottom_app_bar_edit_location": "Modifier localisation",
+  "control_bottom_app_bar_edit_time": "Modifier date & heure",
```
Comment (martabal) [inline mobile/assets/i18n/fr-FR.json:146]:
```suggestion
  "control_bottom_app_bar_edit_location": "Modifier la localisation",
  "control_bottom_app_bar_edit_time": "Modifier la date & l'heure",
```

File: mobile/assets/i18n/fr-FR.json [hunk 5]
```diff
@@ -104,16 +104,16 @@
...
   "backup_manual_success": "Succès ",
   "backup_manual_title": "Statut du téléchargement ",
   "cache_settings_album_thumbnails": "Miniatures de la page bibliothèque ({} éléments)",
   "cache_settings_clear_cache_button": "Effacer le cache",
   "cache_settings_clear_cache_button_title": "Efface le cache de l'application. Cela aura un impact significatif sur les performances de l'application jusqu'à ce que le cache soit reconstruit.",
-  "cache_settings_duplicated_assets_clear_button": "CLEAR",
-  "cache_settings_duplicated_assets_subtitle": "Photos and videos that are black listed by the app",
-  "cache_settings_duplicated_assets_title": "Duplicated Assets ({})",
+  "cache_settings_duplicated_assets_clear_button": "EFFACER",
+  "cache_settings_duplicated_assets_subtitle": "Photos et videos qui sont écartées par l'application",
```
Comment (martabal) [inline mobile/assets/i18n/fr-FR.json:115]:
```suggestion
  "cache_settings_duplicated_assets_subtitle": "Photos et videos qui sont exclues par l'application",
```

File: mobile/assets/i18n/fr-FR.json [hunk 6]
```diff
@@ -370,17 +370,17 @@
...
+  "shared_album_section_people_title": "PERSONNES",
+  "share_dialog_preparing": "Préparation…",
   "shared_link_app_bar_title": "Liens partagés",
-  "shared_link_clipboard_copied_massage": "Copied to clipboard",
-  "shared_link_clipboard_text": "Link: {}\nPassword: {}",
+  "shared_link_clipboard_copied_massage": "Copié dans le presse-papier",
+  "shared_link_clipboard_text": "Lien : {}\nMot de passe : {}",
   "shared_link_create_app_bar_title": "Créer un lien pour partager",
-  "shared_link_create_error": "Error while creating shared link",
+  "shared_link_create_error": "Erreur pendans la création du lien partagé",
```
Comment (martabal) [inline mobile/assets/i18n/fr-FR.json:383]:
```suggestion
  "shared_link_create_error": "Erreur pendant la création du lien partagé",
```

File: mobile/assets/i18n/fr-FR.json [hunk 7]
```diff
@@ -443,10 +443,10 @@
   "trash_page_delete": "Supprimer",
   "trash_page_delete_all": "Tout supprimer",
   "trash_page_empty_trash_btn": "Vider la corbeille",
-  "trash_page_empty_trash_dialog_content": "Voulez-vous vider les éléments de la corbeille? Ces objets seront définitivement retirés d'Immich",
+  "trash_page_empty_trash_dialog_content": "Voulez-vous vider les éléments de la corbeille ? Ces objets seront définitivement retirés d'Immich",
   "trash_page_empty_trash_dialog_ok": "Ok",
   "trash_page_info": "Les éléments mis à la corbeille seront définitivement supprimés au bout de {} jours.",
-  "trash_page_no_assets": "Pas d'éléments dans la corbeille",
+  "trash_page_no_assets": "Pas d'élément dans la corbeille",
```
Comment (martabal) [inline mobile/assets/i18n/fr-FR.json:449]:
```suggestion
  "trash_page_no_assets": "Aucun élément dans la corbeille",
```

File: mobile/assets/i18n/fr-FR.json (hunk 1 above)
Comment (martabal) [reply mobile/assets/i18n/fr-FR.json:272]: I think `actif` is not really a good translation of `asset`. I much prefer to use `media` and there are a lot of references to `asset` in the translation

File: mobile/assets/i18n/fr-FR.json (hunk 4 above)
Comment (martabal) [reply mobile/assets/i18n/fr-FR.json:146]: Maybe change `&` to `et`. It's not really common to see an ampersand in a french text.

Comment (martabal) [reply mobile/assets/i18n/fr-FR.json:146]: From https://fr.wikipedia.org/wiki/Esperluette, "Its use in French is essentially limited to commercial and advertising use, for social reasons."

File: mobile/assets/i18n/fr-FR.json (hunk 2 above)
Comment (axgdcode) [reply mobile/assets/i18n/fr-FR.json:314]: Android parle de photos animées : https://support.google.com/googlecamera/answer/9937175?hl=fr

**Review by martabal — APPROVED** [review]

**Pull request merged**